src/core_modules/capture-core/components/WidgetEventEdit/EditEventDataEntry/editEventDataEntry.actions.js

```diff
@@ -68,3 +69,6 @@ export const startDeleteEventDataEntry = (eventId: string, params: Object) =>
         },
     });
 
+
+export const startCreateNewAfterCompleting = ({ enrollmentId, isCreateNew, orgUnitId, programId, teiId }: Object) =>
```

Comment:
Hey, 
Any reason why `availableProgramStages` is missing from both the action parameters and the `actionCreator` function below? 
Thanks!